com.microsoft.mrtk.tools/SubsystemWizard/Templates/SubsystemConfigTemplate.txt

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do we want to insert Copyright MSFT on new templates? Shouldn't it technically be `Copyright (c) %COMPANYNAME%` or something like that?

Do we even want to automatically apply copyright notices to people's code, even if it's their organization name? Sort of gives a strange impression ("you write an extension to MRTK? COPYRIGHT IT NOW!")

Reply:
i debated on that one.... i think i'll remove them :)